src/DeviceListener.ts

```diff
@@ -319,8 +323,10 @@ export default class DeviceListener {
             for (const device of devices) {
                 if (device.deviceId === cli.deviceId) continue;
 
-                const deviceTrust = await cli.checkDeviceTrust(cli.getUserId()!, device.deviceId!);
-                if (!deviceTrust.isCrossSigningVerified() && !this.dismissed.has(device.deviceId)) {
+                const deviceTrust = await cli
+                    .getCrypto()
+                    ?.getDeviceVerificationStatus(cli.getUserId()!, device.deviceId!);
+                if (!deviceTrust?.crossSigningVerified && !this.dismissed.has(device.deviceId)) {
```

Comment:
The `.getCrypto()?.` here is peculiar, because if there's no crypto it'll add the `deviceId` to `oldUnverifiedDeviceIds`

Reply:
```suggestion
                const deviceTrust = await cli
                    .getCrypto()
                    !.getDeviceVerificationStatus(cli.getUserId()!, device.deviceId!);
                if (!deviceTrust?.crossSigningVerified && !this.dismissed.has(device.deviceId)) {
```